src/libraries/System.Text.Encoding.CodePages/ref/System.Text.Encoding.CodePages.csproj

```diff
@@ -1,9 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Nullable>enable</Nullable>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0</TargetFrameworks>
```

Comment:
While the changes on this project file seem fine to me, I assume that we probably don't want to ship the netcoreapp configuration in the package given that it is already shipped inbox right? If so, we would have to add some extra property on this project file as well as the src project to make sure that this configuration doesn't ship on the package and only ships inbox.

Reply:
Actually seems like the property is already there in the src project so you definitely want to add it here:
```xml
<ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
```

Reply:
Talked offline to @joperezr and decided not to change anything here because it is taken care of by the existing logic mentioned here 
https://github.com/dotnet/runtime/blob/fc15e38da07e26dc2790bf067bd9d7ec17cf44e9/src/libraries/System.Text.Encoding.CodePages/pkg/System.Text.Encoding.CodePages.pkgproj#L33